src/patternfly/components/Drawer/docs/code.md

```diff
@@ -0,0 +1,11 @@
+## Usage
+
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-drawer` | `<div>` | Creates the drawer container. |
+| `.pf-c-drawer__main` | `<div>` | Creates the drawer main container. |
+| `.pf-c-drawer__panel` | `<aside>` | Creates the drawer panel container. |
+| `.pf-c-drawer__panel-body` | `<div>` | Creates the drawer panel body container. |
+| `.pf-m-open` | `.pf-c-drawer` | Modifies the drawer for the open state. |
```

Comment:
Why did you switch `pf-m-expanded` to `pf-m-open`? I liked expanded because its consistent with the naming for many of our other modifiers.